src/pages/additional-features/simple-scripts.mdx

```diff
@@ -0,0 +1,7 @@
+# Simple Scripts
```

Comment:
```suggestion
# Simple Scripts in Atlas: Native and Timelock Scripts in Cardano
```

Reply:
I'd just prefer "Simple Scripts", why ".. in Atlas"? And Native/Timelock are just other names of simple scripts.